manifest.toml

```diff
@@ -42,8 +42,8 @@ ram.runtime = "1G"
 
 [resources]
         [resources.sources.main]
-        url = "https://github.com/Chocobozzz/PeerTube/releases/download/v6.0.3/peertube-v6.0.3.tar.xz"
-        sha256 = "9a88899a4777c340df41e763ecdf8656edf3ef75f54b4de9be05aa309697a211"
+        url = "https://github.com/Chocobozzz/PeerTube/archive/refs/tags/v6.0.4.tar.gz"
```

Comment:
Gotcha, this causes the installation failure and it should be this instead:
https://github.com/Chocobozzz/PeerTube/releases/download/v6.0.4/peertube-v6.0.4.tar.xz